etl/naaccr_etl_sqlserver.sql

```diff
@@ -895,50 +895,49 @@ CREATE TABLE naaccr_data_points_temp
   SELECT ( CASE WHEN  (SELECT MAX(episode_id) FROM episode_temp) IS NULL THEN 0 ELSE
       (SELECT MAX(episode_id) FROM episode_temp) END + row_number() over(order by ndp.record_id))                 AS episode_id
       , ndp.person_id                                                                                                                                             AS person_id
-      , 32531 -- Treatment regimen
+      , ndp.variable_concept_id  -- 32531 Treatment regimen
       , CAST(ndp_dates.naaccr_item_value as date)  		                                                          AS episode_start_datetime        --?
       -- Placeholder... TODO:better universal solution for isnull?
-	  , CASE WHEN LEN(end_dates.naaccr_item_value) > 1
+	    , CASE WHEN CHAR_LENGTH(end_dates.naaccr_item_value) > 1
```

Comment:
Need to switch this back to LEN for sqlserver version. LEN converts to CHAR_LENGTH